AUTHORS

```diff
@@ -3361,5 +3361,12 @@ Francesco Trotta <github@fasttime.org>
 Richie Bendall <richiebendall@gmail.com>
 ryan <chai_xb@163.com>
 Piotr Rybak <rybak.piotr@yahoo.com>
+Jean Burellier <sheplu@users.noreply.github.com>
+Subhi Al Hasan <subhialhasan@protonmail.com>
+3nprob <3nprob@3nprob>
```

Comment:
Not technically a valid email address. Not sure if this is a problem @Trott.

Reply:
I don't think it's a problem. There's already a similarly-formatted non-email at https://github.com/nodejs/node/blob/afe460ec9e30add944056760a01d1185c6aac276/AUTHORS#L1993. 

All that said, if @3nprob wants their identity in the AUTHORS file to be expressed differently, we can certainly do that.

Reply:
That would be great actually. Just made a suggestion with a valid e-mail